src/status_im2/contexts/syncing/events.cljs

```diff
@@ -62,15 +61,11 @@
 
 (rf/defn preparations-for-connection-string
   {:events [:syncing/get-connection-string-for-bootstrapping-another-device]}
-  [{:keys [db]} entered-password]
+  [{:keys [db]} entered-password set-code]
   (let [valid-password? (>= (count entered-password) constants/min-password-length)
         show-sheet      (fn [connection-string]
-                          (rf/dispatch
-                           [:show-bottom-sheet
-                            {:content (fn []
-                                        [sheet/qr-code-view-with-connection-string
-                                         connection-string])}])
-                          (rf/dispatch [:syncing/update-role constants/local-pairing-role-sender]))]
```

Comment:
@J-Son89 - Please add back the `:syncing/update-role` event dispatch as it's needed for successful navigation upon syncing completion.

Removing this dispatch will cause this bug: https://github.com/status-im/status-mobile/pull/15574#issuecomment-1498980276